CMakeLists.txt

```diff
@@ -77,14 +77,16 @@ include(cmake/init_target.cmake)
 include(cmake/generate_target.cmake)
 include(cmake/target_yasm_sources.cmake)
 include(cmake/external.cmake)
-include(cmake/libabsl.cmake)
-include(cmake/libopenh264.cmake)
 include(cmake/libpffft.cmake)
 include(cmake/librnnoise.cmake)
 include(cmake/libsrtp.cmake)
-include(cmake/libusrsctp.cmake)
-include(cmake/libvpx.cmake)
-include(cmake/libyuv.cmake)
+if (NOT TG_OWT_PACKAGED_BUILD)
```

Comment:
This breaks the fallback logic when a library is not found

Reply:
Yes, that can't be right.
https://github.com/desktop-app/tg_owt/pull/85 should do the trick, though.

Reply:
Yes, this is wrong, as Ilya has already told me. 598697567eaa46aec7589b81689cc836ff99e4a4 should fix that.
(However, there are linking problems with snap pack builds there; I haven't had a chance to reproduce it yet, but I'm still going to)